pkg/common/config/types.go

```diff
@@ -75,10 +75,18 @@ type Config struct {
 	// Guest Cluster configurations, only used by GC
 	GC GCConfig
 
-	// Tag categories and tags which correspond to "built-in node labels: zones and region"
+	// Labels will list the topology domains the CSI driver is expected
+	// to pick up from the inventory. This info will later be used while provisioning volumes.
 	Labels struct {
-		Zone   string `gcfg:"zone"`
-		Region string `gcfg:"region"`
+		// Zone and Region correspond to the vSphere categories
+		// created to tag specific topology domains in the inventory.
+		Zone   string `gcfg:"zone"`   // Deprecated
```

Comment:
Should we deprecate these parameters? We want new customers to use `topologycategories` parameter for zone and region as well but existing customers will have to delete and re-create nodes on their cluster to use `topologycategories` parameter for zone and region as they will be applied with a different label name through this parameter. They will also need to change their SC `allowedTopologies` value and PV nodeAffinity rules.

Reply:
yes, `Deprecated` comment beside the older config field is good.

Reply:
Okay, we will mention in the upgrade documentation that `zone` and `region` will be supported in the foreseeable future but new customers should use `topology-categories`.